src/task1.py

```diff
@@ -0,0 +1,23 @@
+"""List practice"""
+
+# 1
+list1 = [ltr1 + ltr2 for ltr1 in ['a', 'b'] for ltr2 in ['b', 'c', 'd']]
+print(list1)
+
+# 2
+list2 = list1[::2]
+print(list2)
+
+# 3
+list3 = [str(x) + 'a' for x in range(1, 5)]
+print(list3)
+
+# 4
+print(list3.pop(1))
+print(list3)
+
+# 5
+list4 = list3.copy()
+list4.append('2a')
```

Comment:
it would be in the end

Reply:
Yes, in task description it wasn't explicitly declared that `2a` should be added to its original position